.travis.yml

```diff
@@ -1,33 +1,37 @@
 language: php
-
-sudo: required
+os: linux
 
 services:
   - docker
 
 php:
-  - '5.6'
-  - '7.0'
-  - '7.1'
   - '7.2'
   - '7.3'
-  - hhvm
+  - '7.4'
   - nightly
 
 env:
-  - MYSQL_VERSION=5.7 PG_VERSION=9.1
-  - MYSQL_VERSION=8.0 PG_VERSION=9.2
-  - MARIA_VERSION=5.5 PG_VERSION=9.3
-  - MARIA_VERSION=10.0 PG_VERSION=9.4
-  - MARIA_VERSION=10.1 PG_VERSION=9.5
-  - MARIA_VERSION=10.2 PG_VERSION=9.6
-  - MARIA_VERSION=10.3 PG_VERSION=10
-  - MARIA_VERSION=10.4 PG_VERSION=11
+  # Laravel 5.* Jobs
+  - MYSQL_VERSION=5.7 PG_VERSION=9.1 LA_VERSION="5.*"
+  - MYSQL_VERSION=8.0 PG_VERSION=9.2 LA_VERSION="5.*"
+  - MARIA_VERSION=5.5 PG_VERSION=9.3 LA_VERSION="5.*"
+  - MARIA_VERSION=10.0 PG_VERSION=9.4 LA_VERSION="5.*"
+  - MARIA_VERSION=10.1 PG_VERSION=9.5 LA_VERSION="5.*"
+  - MARIA_VERSION=10.2 PG_VERSION=9.6 LA_VERSION="5.*"
+  - MARIA_VERSION=10.3 PG_VERSION=10 LA_VERSION="5.*"
+  - MARIA_VERSION=10.4 PG_VERSION=11 LA_VERSION="5.*"
```

Comment:
Might be good to set these to `5.5.*`, so we can ensure it's not just `5.8.*` that passes...